Samples/GitHubLabeler/.gitignore

```diff
@@ -0,0 +1,288 @@
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+##
```

Comment:
I don't think checking in this .gitignore file was intentional, was it?

Reply:
Agreed. We should only use the one from the root.